src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/__tests__/Mock Objects/MockExposureSubmissionService.swift

```diff
@@ -68,6 +68,8 @@ class MockExposureSubmissionService: ExposureSubmissionService {
 	var devicePairingConsentAcceptTimestamp: Int64?
 
 	var devicePairingSuccessfulTimestamp: Int64?
+	
+	var isAllowedToAutomaticallyShareTestResults = false
```

Comment:
I like that this name is very explicit, but I guess it could be more consistent with the service itself. The method that we are allowed to automatically call is named `submitExposure`, and it is the `ExposureSubmissionService`. So it could be called `isAllowedToAutomaticallySubmitExposure`. Another shorter suggestion would be `submissionConsentGiven` or even just `consentGiven`. I think I like these shorter variants even a bit more 😁

Reply:
i also like submissionConsentGiven

Reply:
And the winner is 🥁 : `submissionConsentGiven`

Reply:
Was denkt ihr darüber, wenn wir noch _automatic_ mit aufnehmen?